packages/neo-one-website/src/components/reference/ReferenceGrid.tsx

```diff
@@ -0,0 +1,58 @@
+// tslint:disable no-any
+import { TextInput } from '@neo-one/react-common';
+import * as React from 'react';
+import { Grid, styled } from 'reakit';
+import { ReferenceFilterContainer } from '../../containers';
+import { ReferenceLink } from './ReferenceLink';
+import { TypeFilter } from './TypeFilter';
+import { ReferenceItem } from './types';
+
+const ReferenceLayout = styled(Grid)`
+  min-width: 0;
+  grid-template-columns: repeat(auto-fill, 160px);
+`;
+
+const Wrapper = styled(Grid)`
+  gap: 16px;
+  min-width: 0;
+`;
+
+const StyledInput = styled(TextInput)`
+  height: 36px;
```

Comment:
Any particular reason for using 36? Generally we want to stick to multiples of 8.